ilastik/applets/objectExtraction/opObjectExtraction.py

```diff
@@ -576,7 +588,27 @@ def _augmentFeatureNames(self, features):
 
         return feature_names_with_default
 
-    def _extract(self, image, labels):
+    def _createAtlasMapping(self, region_centers:numpy.ndarray, atlasImage:numpy.ndarray):
+        axes = {tag.key:idx for idx, tag in  enumerate(atlasImage.axistags)}
+        num_center_points = len(region_centers)
+        num_channels_in_atlas = atlasImage.shape[axes['c']] if 'c' in axes else 1
+        atlas_mapping = numpy.zeros(num_center_points * num_channels_in_atlas)
```

Comment:
```suggestion
        atlas_mapping = numpy.zeros((num_center_points, num_channels_in_atlas))
```
will save you a line (generates the array in the right shape)